src/phockup.py

```diff
@@ -128,24 +128,32 @@ def walk_directory(self):
 
         # Walk the directory
         for root, dirnames, files in os.walk(self.input_dir):
-            files.sort()
-            file_paths_to_process = []
-            for filename in files:
-                if filename in ignored_files:
-                    continue
-                file_paths_to_process.append(os.path.join(root, filename))
-            if self.max_concurrency > 1:
-                if not self.process_files(file_paths_to_process):
-                    return
+            logger.info(f"walking directory. current dir: {root}, dirnames: {dirnames}, files: {files}")
+            root_set = set(root.split(os.sep))
+            intersect_set = ignored_dirs.intersection(root_set)
+            logger.info(f"root_set: {root_set}, ignored_set: {ignored_dirs}, intersection: {intersect_set}")
```

Comment:
`ignored_set` and `intersection` are more technical details. I belive the log should actually say what are these used for. For example `ignored directories:`.